lldb/include/lldb/Core/Progress.h

```diff
@@ -97,6 +119,10 @@ class Progress {
   /// The title of the progress activity.
   std::string m_title;
   std::mutex m_mutex;
+  /// Set to true if the progress event is aggregate; meaning it will happen
+  /// multiple times during a debug session as individual progress events
```

Comment:
comment is out of date, it mentions "Set to true ..."

Reply:
Noted, I'll change this 👍🏾